bundled_scripts/ilo.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+import os
+import sys
+from copy import deepcopy
+
+import hpilo
+
+
+USER = os.environ['MANAGEMENT_USER_NAME']
+PASS = os.environ['MANAGEMENT_USER_PASSWORD']
+
+MAC_PREFIX_BLACKLIST = [
+    '505054', '33506F', '009876', '000000', '00000C', '204153', '149120',
+    '020054', 'FEFFFF', '1AF920', '020820', 'DEAD2C', 'FEAD4D',
+]
+
+DEVICE_INFO_TEMPLATE = {
```

Comment:
In general, yes, I think it's a good idea. But IMO it's too soon for such optimizations - there are only two simple scripts here, and not enough common stuff between them to justify the existence of a separate package.